### PR TITLE
fix:  real full screen for nft img #7846

### LINKED ISSUE #7846: Can we make fullscreen to be real full screen and not full window?
Right now, it does full-window, not full-screen.

https://github.com/kodadot/nft-gallery/assets/5887929/6924d1dc-d0b3-42bc-991a-e876a24b5cdc

### PR DESCRIPTION
**Pull request by shengxinjing**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Context

- [x] Closes #7846
- [ ] Requires deployment <snek/rubick/worker>

#### Did your issue had any of the "$" label on it?

- [ ] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=<My_Polkadot_Address_check_https://github.com/kodadot/nft-gallery/blob/main/REWARDS.md#creating-your-dot-address>)

### <samp>🤖 Generated by Copilot at 2b09d77</samp>

The pull request enhances the user experience of viewing media items in the activity page by adding a fullscreen mode option. It also removes some obsolete code and components and adjusts the layout margin. The main files affected are `components/gallery/GalleryItem.vue` and `components/collection/activity/Activity.vue`.

### <samp>🤖 Generated by Copilot at 2b09d77</samp>

> _`useFullscreen` hook_
> _media items fill the screen_
> _winter solitude_

### CODE REVIEW

**Comment by exezbcz** [comment]
nice!

for not-square imgs it does not work and the img does not fit the fullscreen 
image


https://deploy-preview-7860--koda-canary.netlify.app/ksm/gallery/20237759-7EA1DCF47E98A25067-RDNWYGCTFOERX-DESERT_VILLA_2-00000005

**Comment by daiagi** [comment]
please use 1 of the PR templates

**Comment by yangwao** [comment]
after clicking fullscreen I see only part of image

Screenshot 2023-10-28 at 12 49 36

image

**Comment by shengxinjing** [comment]
@exezbcz @yangwao fixed
image

**Comment by exezbcz** [comment]
looks good! one last thing is the dark mode 

image

- can you change the background to dark mode color

**Comment by shengxinjing** [comment]
> looks good! one last thing is the dark mode
> 
> image
> 
> * can you change the background to dark mode color

fixed

**Comment by exezbcz** [comment]
yup, works for me, mobile works too

thanks!

**Comment by prury** [comment]
> yup, works for me, mobile works too
> 
> thanks!

here mobile show some strange lines:
other than that it works 
Imagem do WhatsApp de 2023-10-30 à(s) 10 03 33_b6515307

File: components/gallery/GalleryItem.vue [hunk 1]
```diff
@@ -6,19 +6,23 @@
       :subtitle="$t('mint.successCreateNewNft', [congratsNewNft])" />
     <div class="columns is-variable is-6">
       <div class="column is-two-fifths">
-        <div class="is-relative">
+        <div id="nft-img-container" ref="imgref" class="is-relative">
```
Comment (daiagi) [inline components/gallery/GalleryItem.vue:9]: we don't really use ids in the code base
is it necessary here?
prefer class

File: components/gallery/GalleryItem.vue [hunk 2]
```diff
@@ -209,18 +208,15 @@ const tabs = {
 }
 const activeTab = ref(tabs.offers)
 
-const isFullscreen = ref(false)
+// const isFullscreen = ref(false)
```
Comment (daiagi) [inline components/gallery/GalleryItem.vue:211]:
```suggestion
```

File: components/gallery/GalleryItem.vue [hunk 3]
```diff
@@ -315,7 +321,15 @@ $break-point-width: 930px;
     }
   }
 }
-
+.back-button {
+  position: absolute;
+  left: 0.75rem;
+  top: 2rem;
+  z-index: 99;
```
Comment (daiagi) [inline components/gallery/GalleryItem.vue:328]: why this z-index?

File: components/gallery/GalleryItem.vue (hunk 1 above)
Comment (shengxinjing) [reply components/gallery/GalleryItem.vue:9]: I need to use the priority of the id selector to override the style when it's not fullscreen

File: components/gallery/GalleryItem.vue (hunk 3 above)
Comment (shengxinjing) [reply components/gallery/GalleryItem.vue:327]: I need to set the z-index to be able to overlay on top of the global image, but it doesn't need to be as big as 99. Setting it to 1 works well enough

**Review by daiagi — APPROVED** [review]

**Review by preschian — APPROVED** [review]

**Comment by yangwao** [comment]
thanks !

**Pull request merged**